compiler/rustc_resolve/src/lib.rs

```diff
@@ -1426,7 +1440,7 @@ impl<'a> Resolver<'a> {
         Default::default()
     }
 
-    pub fn into_outputs(self) -> ResolverOutputs {
+    pub fn into_outputs(self) -> (Definitions, Box<CrateStoreDyn>, ResolverOutputs) {
```

Comment:
Why is it necessary to move definitions and crate store from `ResolverOutputs` though?
`ResolverOutputs` are dismantled anyway when tcx is created, so what is the point in partially dismantling it slightly earlier?